pkg/sdk/go/sdk.go

```diff
@@ -370,12 +370,17 @@ type SDK interface {
 	//  fmt.Println(id)
 	IdentifyThing(key string) (string, errors.SDKError)
 
-	// ShareThing shares thing with other user.
+	// ShareThing shares thing with other user. It assumes that you have
+	// already created a group and added things to it. It also assumes that
+	// you have required policy to share a thing with the specified user.
+	//
+	// Under the hoold `ShareThing` calls the `Connect` method with the
```

Comment:
Typo `hood`.  But no need for that at all: `It calls Connect method with...` is sufficient.